src/api.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```

Comment:
- [ ] Linters help us write better code. Kindly enable linters here.

Reply:
solved, the file in question was unnecessary, but I forgot to delete it.